### PR TITLE
Setup for Streamlit

### PR DESCRIPTION
**Pull request by The-Pavel**
Hey!
* Added a `Procfile`
* Added a `app.py` with minimal setup

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: streamlit run app.py
```
Comment (The-Pavel) [inline Procfile]: Hey Pavel - looks like you forgot the `setup.sh` for Heroku

File: app.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import streamlit as st
+
+st.write('Hello world')
```
Comment (The-Pavel) [inline app.py:3]: LGTM!

**Review by phaberman — COMMENTED** [review]
lvgtm

File: Procfile (hunk 1 above)
Comment (The-Pavel) [reply Procfile]: Ah damn, let me add a setup.sh

File: app.py (hunk 2 above)
Comment (phaberman) [reply app.py:3]: LVGTM!

**Pull request merged**